### ISSUE TITLE
More maintainers

### ISSUE THREAD

**Issue by theoephraim** [NYTimes/backbone.trackit#25]
I love using trackit but have run into some issues and it appears others have as well.
There are a bunch of forks, issues, and PRs.

Need some help?

**Comment by delambo** [comment]
Yes, and sorry for the late reply. If you or anyone would like to be a maintainer, please comment on some issues, create some pull requests or handle other maintenance around the repository. When I see some action I'll gladly add you to the NYTimes/backone.trackit org.
